pkg/config/config.go

```diff
@@ -38,6 +38,7 @@ type SharedSystemSettings struct {
 type SharedSettings struct {
 	Snapshotters []string `yaml:"snapshotters,omitempty"`
 	CredsHelpers []string `yaml:"creds_helpers,omitempty"`
+	Mode         *string  `yaml:"mode,omitempty"`
```

Comment:
I wonder if `mode` is too generic of a name here. Maybe `dockercompat: true` makes more sense?

Reply:
Acknowledged!
Changing in next revision...